ftplugin/toml.vim

```diff
@@ -10,24 +10,10 @@ let b:did_ftplugin = 1
 
 let s:save_cpo = &cpo
 set cpo&vim
+let b:undo_ftplugin = 'setlocal commentstring< comments<'
 
 setlocal commentstring=#\ %s
-
-" Add NERDCommenter delimiters
-
-let s:delims = { 'left': '#' }
-if exists('g:NERDDelimiterMap')
-    if !has_key(g:NERDDelimiterMap, 'toml')
-        let g:NERDDelimiterMap.toml = s:delims
-    endif
-elseif exists('g:NERDCustomDelimiters')
-    if !has_key(g:NERDCustomDelimiters, 'toml')
-        let g:NERDCustomDelimiters.toml = s:delims
-    endif
-else
-    let g:NERDCustomDelimiters = { 'toml': s:delims }
-endif
-unlet s:delims
+setlocal comments=:#
```

Comment:
Why is the `:` here?

Reply:
That's how the `comments` option works. From the vim help:

> The 'comments' option is a comma-separated list of parts.  Each part defines a
> type of comment string.  A part consists of:
>     {flags}:{string}
> ...
> When a string has none of the 'f', 's', 'm' or 'e' flags, Vim assumes the
> comment string repeats at the start of each line.  The flags field may be
> empty.

Reply:
Thanks! Looks like `format-comments` was the key section.